src/_includes/website_navigation_sub_menus.liquid

```diff
@@ -0,0 +1,17 @@
+<ul 
+  class="website-navigation-sub-menus__menu" 
```

Comment:
Should we change this to a BEM block now?

Reply:
Yep! I'd been meaning to come back for this: https://github.com/codurance/site/pull/1921/commits/76a41f18aabaa276344efa5a681250c0b45224d5